Getting the error message "Failed to execute goal io.fabric8:docker-maven-plugin:0.28.0:build"

When building the ETS, the following error message occurs.

```
[ERROR] Failed to execute goal io.fabric8:docker-maven-plugin:0.28.0:build (build) on project ets-ogcapi-tiles10: Execution build of goal io.fabric8:docker-maven-plugin:0.28.0:build failed: An API incompatibility was encountered while executing io.fabric8:docker-maven-plugin:0.28.0:build: java.lang.UnsatisfiedLinkError: could not load FFI provider jnr.ffi.provider.jffi.Provider
[ERROR] -----------------------------------------------------
[ERROR] realm =    plugin>io.fabric8:docker-maven-plugin:0.28.0
[ERROR] strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
```

**Environment:**

* Java 11
* Maven 3.8.8

--- a/pom.xml
+++ b/pom.xml
@@ -271,7 +271,7 @@
         <plugin>
           <groupId>io.fabric8</groupId>
           <artifactId>docker-maven-plugin</artifactId>
-          <version>0.28.0</version>
+          <version>0.45.0</version>
           <configuration>
             <images>
               <image>

